zboss/doc/zboss_configuration.rst

```diff
@@ -8,67 +8,21 @@ ZBOSS library configuration
    :depth: 2
 
 The ZBOSS stack is distributed as a set of precompiled linkable libraries.
-These libraries are available in the following versions:
 
-* `ZBOSS production libraries`_
-* `ZBOSS development libraries`_
-
-Each version contains variants of the libraries that support different Zigbee device roles, each variant having its own configuration.
+The library support different Zigbee device roles, each variant having its own configuration.
 This allows you to scale the application and select the most suitable set of features.
 
-The following table describes the available Zigbee libraries and supported configurations.
-
-.. list-table::
-   :header-rows: 1
-
-   * - Library
-     - nRF5340 single SoC
-     - nRF5340 NCP
-     - nRF52833 single SoC
-     - nRF52833 NCP
-     - nRF52840 single SoC
-     - nRF52840 NCP
-   * - Production
-     -
-     -
-     - x
-     -
-     - x
-     -
-   * - Development
-     - x
-     - x
-     - x
-     - x
-     - x
-     - x
-
-ZBOSS production libraries
-**************************
-
-|list_intro|
-
-* :file:`zboss/production/include/osif/libzboss_config.h` - library for Coordinators and Routers
-* :file:`zboss/production/include/osif/libzboss_config.ed.h` - library for End Devices
-
-|list_outro|
-
-The ZBOSS production library version is enabled by default with the :option:`CONFIG_ZIGBEE_LIBRARY_PRODUCTION` Kconfig option.
-The production libraries support only the nRF52840 and nRF52833.
-
-ZBOSS development libraries
+ZBOSS libraries
```

Comment:
ZBOSS device role libraries? Or maybe we don't need this section header at all?

Reply:
removed